packages/strapi-design-system/src/Row/Row.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+export const Row = styled.div`
+  display: flex;
+  flex-direction: ${({ reverse }) => reverse};
+  justify-content: ${({ justifyContent }) => justifyContent};
+  align-items: ${({ alignItems }) => alignItems};
+  flex-wrap: ${({ wrap }) => wrap};
+`;
+
+Row.defaultProps = {
+  reverse: false,
+  justifyContent: undefined,
+  alignItems: 'center',
+  wrap: undefined,
+};
+
+Row.propTypes = {
+  reverse: PropTypes.bool,
+  justifyContent: PropTypes.string,
```

Comment:
Can you order the propTypes alphabetically ?

Reply:
come oooon, right, I can do that 😆

Reply:
Maybe we could add this https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md ?

Reply:
Yes, why not 😊 . Can you do it?